vmdb/spec/models/ems_refresh/refreshers/openstack_refresher_rhos_grizzly_spec.rb

```diff
@@ -67,7 +67,7 @@ def assert_table_counts
     SystemService.count.should       == 0
 
     Relationship.count.should        == 13
-    MiqQueue.count.should            == 19
+    MiqQueue.count.should            == 20
```

Comment:
I'm fine with changing this if we know that's it's correct.  The fact that we're asserting the count of rows in miq_queue is pretty vague.  It's not clear what we're asserting here.  I'd rather change these to assert on specifics types of miq_queue rows.  @blomquisg any idea how we can change this assertion to be more specific?